src/environment.js

```diff
@@ -147,15 +147,15 @@ export class DeclarativeEnvironmentRecord extends EnvironmentRecord {
       deletable: D,
       environment: envRec,
       name: N,
-      isGlobal,
+      isGlobal: !!isGlobal,
```

Comment:
The parameter is already a Boolean, this makes the reader perplexed and confused.